src/modules/users/controller/controller.ts

```diff
@@ -0,0 +1,67 @@
+import { RequestHandler } from 'express';
+
+import { fromRawToLoginSuggestResults, fromRawToUserData } from '../mappers';
+import { User } from '../model';
+import { UserRepository } from '../repository';
+import { checkForExistingLogin, findById } from './helpers';
+import { RequestBodyCreateUser, RequestBodyUpdateUser } from './types';
+
+export const getUser: RequestHandler = async (req, res) => {
+    const { id } = req.params;
+    const userRepo = new UserRepository(User);
+
+    const userRecord = await findById(userRepo, id);
+
+    res.json(fromRawToUserData(userRecord));
+};
+
+export const createUser: RequestHandler = async (req, res) => {
+    const { login, password, age } = req.body as RequestBodyCreateUser;
+    const userRepo = new UserRepository(User);
+
+    await checkForExistingLogin(userRepo, login);
+
+    const newUserRecord = await userRepo.create({
+        login,
+        password,
+        age
+    });
+
+    res.status(201).json(fromRawToUserData(newUserRecord));
+};
+
+export const updateUser: RequestHandler = async (req, res) => {
+    const { id } = req.params;
+    const { login, age } = req.body as RequestBodyUpdateUser;
+    const userRepo = new UserRepository(User);
+
+    await findById(userRepo, id);
+    await checkForExistingLogin(userRepo, login);
+
+    await userRepo.update(id, {
+        login,
+        age
+    });
+
+    res.sendStatus(200);
```

Comment:
update request should return updated user data.

Reply:
done